extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -243,17 +313,22 @@ public function saveSettings( $settings, $cityId = null ) {
 						$history[$i]['settings']['favicon-image-url'] = $oldFaviconFile['url'];
 					}
 				}
-				if ( isset( $oldFile ) && isset( $history[$i]['settings']['wordmark-image-name'] ) ) {
+				if ( isset( $oldWordmarkFile ) && isset( $history[$i]['settings']['wordmark-image-name'] ) ) {
 					if ( $history[$i]['settings']['wordmark-image-name'] == self::WordmarkImageName ) {
-						$history[$i]['settings']['wordmark-image-name'] = $oldFile['name'];
-						$history[$i]['settings']['wordmark-image-url'] = $oldFile['url'];
+						$history[$i]['settings']['wordmark-image-name'] = $oldWordmarkFile['name'];
+						$history[$i]['settings']['wordmark-image-url'] = $oldWordmarkFile['url'];
 					}
 				}
 				if ( isset( $oldBackgroundFile ) && isset( $history[$i]['settings']['background-image-name'] ) ) {
 					if ( $history[$i]['settings']['background-image-name'] == self::BackgroundImageName ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)